scripts/basic-workflow-tests.sh

```diff
@@ -186,6 +186,22 @@ test_train() {
     fi
 
     ilab model train "${TRAIN_ARGS[@]}"
+
+    if [ "$FULLTRAIN" -eq 1 ]; then
+
+        # TODO Only cuda for now
+        ilab config init --non-interactive --train-profile="${SCRIPTDIR}/test-data/train-profile-a10.yaml"
+        TRAIN_ARGS=("--device=cuda" "--model-path=instructlab/granite-7b-lab" "--data-path=${SCRIPTDIR}/test-data/train_all_pruned_SDG.jsonl" "--quantize-data-type=nf4" "--4-bit-quant")
+        if [ "$GRANITE" -eq 1 ]; then
+            TRAIN_ARGS+=("--gguf-model-path models/granite-7b-lab-Q4_K_M.gguf")
+        fi
+
+        ilab model train "${TRAIN_ARGS[@]}"
+
+        ilab config init --non-interactive
```

Comment:
what is this for?

Reply:
resetting the cfg back to the defaults after initializing a train profile

Reply:
... but why :) we only need to train once

Reply:
removed